apps/evm/src/clients/api/queries/useGetPools/getPools/getApiPools/index.ts

```diff
@@ -80,6 +80,7 @@ export interface ApiMarket {
   lastCalculatedXvsAccruedBlockNumber: string | null;
   supplyCapsMantissa: string;
   borrowCapsMantissa: string;
+  badDebt: string;
```

Comment:
I understand this is the schema from the API, but I'd recommend updating it at some point to be consistent with the naming convention: `badDebtMantissa`.